test/run-unit-tests.py

```diff
@@ -17,12 +16,11 @@
 except ImportError:
     GREEN = YELLOW = RED = RESET_ALL = BRIGHT = ''
 
-@dataclasses.dataclass
-class Total:
-    total:int
-    good:int = 0
-    skip:int = 0
-    fail:int = 0
+class total:
```

Comment:
maybe add a FIXME comment suggesting this is switched back to dataclasses one day?